profile.in

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
-
 #
-# Default profile.
+# YADRO OpenBMC Command Line Interface
+# Default profile
+# Copyright (C) 2020-2021 YADRO
+#
+# SPDX-License-Identifier: Apache-2.0
 #
 
 # greeting
 cat << EOF
-Welcome to BMC for $(sed -n 's/OPENBMC_TARGET_MACHINE="\(.*\)"/\1/p' /etc/os-release) server!
-Type 'help' to print the list of useful commands.
+Welcome to BMC Command Line Interface for $(sed -n 's/OPENBMC_TARGET_MACHINE="\(.*\)"/\1/p' /etc/os-release)!
+Copyright (C) 2020-2021 YADRO
```

Comment:
What do you think about `$(date +%Y)` instead of the last year in the banner?

Reply:
It would be good if the local date/time source was 100% reliable. Otherwise if somehow it gets reset/corrupted, we'll end up with a copyright ending in the past. Also, I'm not sure that such an approach is correct from the legal point of view.